pkg/kncloudevents/v2/cloudevents.go

```diff
@@ -63,14 +64,14 @@ var _ cloudevents.Client = (*client)(nil)
 func (c *client) Send(ctx context.Context, out event.Event) error {
 	c.applyOverrides(ctx, &out)
 	err := c.ceClient.Send(ctx, out)
-	return c.reportCount(ctx, out, err)
+	return c.reporter.ReportCount(ctx, out, err)
 }
 
 // Request implements client.Request
 func (c *client) Request(ctx context.Context, out event.Event) (*event.Event, error) {
 	c.applyOverrides(ctx, &out)
```

Comment:
overrides are only for sources.

Reply:
not sure what to do here. Can you expand?